blocks/hero/hero.js

```diff
@@ -0,0 +1,73 @@
+import { toCamelCase } from '../../scripts/aem.js';
+import { div } from '../../scripts/dom-helpers.js';
+
+function readHeroBlockConfig(block) {
+  const config = {};
+
+  block.querySelectorAll(':scope > div').forEach((row) => {
+    if (row.children) {
+      const cols = [...row.children];
+      if (cols[1]) {
+        const col = cols[1];
+        const name = toCamelCase(cols[0].textContent);
+        config[name] = col.children || [];
+      }
+    }
+  });
+  return config;
+}
+
+export default function decorate(block) {
+  const heroConfig = readHeroBlockConfig(block);
+  let mobileImage = heroConfig.mobileImage ? heroConfig.mobileImage[0] : null;
+  let desktopImage = heroConfig.desktopImage ? heroConfig.desktopImage[0] : null;
+  const image = heroConfig.image ? heroConfig.image[0] : null;
+
+  if (mobileImage && desktopImage) {
+    mobileImage.classList.add('mobile-image');
+    desktopImage.classList.add('desktop-image');
+  } else {
+    mobileImage = mobileImage || desktopImage || image;
+    desktopImage = desktopImage || mobileImage || image;
+  }
+
+  block.innerHTML = '';
+  block.appendChild(
+    div({ class: 'grid' },
+      div({ class: 'hero-text' },
+        ...heroConfig.text,
+      ),
+      div({ class: 'hero-image' },
+        mobileImage, // mobile first always for LHS
+        desktopImage,
+      ),
+    ),
+  );
+
+  const heroText = block.querySelector('.hero-text');
+  const heroImage = block.querySelector('.hero-image');
+
+  if (block.classList.contains('standard-hero-overlap')) { // should this the default?
+    heroText.classList.add(...'g-col-xl-6 g-col-lg-7 g-col-md-7 g-col-12'.split(' '));
```

Comment:
Why don't we just use `heroText.classList.add('g-col-xl-6', 'g-col-lg-7', 'g-col-md-7', 'g-col-12');`? For all occurences.